Automattic/jetpack
# Contact Form: inherit font size for input fields

Fixes #23729

#### Changes proposed in this Pull Request:

Since we do not define a custom font size for our input fields, let's ensure we inherit it from other values set in themes, like we do for other field types such as textarea.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

This should be tested on a variety of themes: block-based (Twenty Twenty Two, Livro, ...), non block-based default themes (Twenty Twenty), non block-based popular themes (Astra, OceanWP).

* Go to Jetpack > Settings and ensure the Contact form feature is active.
* Go to Posts > Add New and insert a form with input fields (the default contact form will work)
* Publish post, and visit frontend; the font size in the input fields should look okay.
